lib/mosaic-interactions/src/main/java/org/eclipse/mosaic/interactions/vehicle/VehicleDistanceSensorActivation.java

```diff
@@ -43,36 +43,52 @@
         FRONT, LEFT, RIGHT, REAR
     }
 
+    /**
+     * Sensor type.
+     */
+    private final SensorTypes sensorType;
+
     /**
      * String identifying a simulated vehicle.
      */
     private final String vehicleId;
+    /**
+     * list of distance sensors.
+     */
+    private final DistanceSensors[] sensors;
 
     /**
      * The maximum distance to look ahead for a leading vehicle.
      */
     private final double maximumLookahead;
-
     /**
-     * list of distance sensors.
+     * True if sensor shall be spawned, False if it shall be destroyed.
      */
-    private DistanceSensors[] sensors;
+    private final boolean activate;
 
     /**
      * Creates a new {@link VehicleDistanceSensorActivation} interaction.
      *
      * @param time             Timestamp of this interaction, unit: [ns]
      * @param vehicleId        vehicle identifier
+     * @param sensorType
      * @param maximumLookahead maximum distance to look ahead for a leading vehicle
+     * @param activate
      * @param sensors          list of distance sensors
      */
-    public VehicleDistanceSensorActivation(long time, String vehicleId, double maximumLookahead, DistanceSensors... sensors) {
+    public VehicleDistanceSensorActivation(long time, String vehicleId, SensorTypes sensorType, double maximumLookahead, boolean activate, DistanceSensors... sensors) {
```

Comment:
due to the merge with the Lidar activation this is a bit messy now. I would've imagine following:

* [ ] rename the class to `VehicleSensorActivation`
* [ ] instead of `DistanceSensors` and `SensorTypes` enums use a new enum `SensorType` with values `LIDAR`, `RADAR_FRONT`, `RADAR_REAR`, `RADAR_LEFT`, and `RADAR_RIGHT`
* [ ] use `maximumLookahead` instead of `activate` flag to enable lidar sensor in Carla